[QRadar v3] Metrics (#34485)
demisto/content: Packs/QRadar/Integrations/QRadar_v3/README.md

If the command takes too long to finish executing, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).

### API Call Metrics

> This feature provides support for tracking the QRadar API calls.

Reviewer: ```suggestion
This feature collects metadata on QRadar API calls and their success status.
```

> This feature provides support for tracking the  QRadar API calls.

Author: ```suggestion
This feature provides support for tracking the QRadar API calls.
```

> Please note that this feature does not include metrics for long-running commands such as `fetch incidents`.

Reviewer: ```suggestion
API Call metrics are not available for long-running commands such as `fetch incidents`.
```

> To view the API Call Metrics, navigate to the **Dashboard & Reports** section in the side-bar and than to the **API Execution Metrics** dashboard.

Reviewer: ```suggestion
API Metrics are shown in the built-in **API Execution Metrics** dashboard, and are available to use in custom widgets.
```
Please run it by a tech writer

Reviewer 2: From tech writer: Looks good.
